app/src/main/java/de/westnordost/streetcomplete/quests/crossing/AddCrossingForm.kt

```diff
@@ -12,11 +12,10 @@ class AddCrossingForm : AListQuestForm<CrossingAnswer>() {
     private val mapDataSource: MapDataWithEditsSource by inject()
 
     override val items = listOf(
-        TextItem(YES, R.string.quest_crossing_yes),
-        TextItem(NO, R.string.quest_crossing_no),
+        TextItem(YES, R.string.quest_crossing_yes2),
+        TextItem(INFORMAL, R.string.quest_crossing_no2),
```

Comment:
Can I simply bump the string id here or do I need to implement a fallback for translations that do not yet contain the new string? (Something like "if quest_crossing_yes2 does not exist, then use quest_crossing_yes")